src/main.c

```diff
@@ -20,24 +20,33 @@ int main(int argc, char *argv[])
     // print_parameters();
 
     populacao *result;
+    populacao *population = NULL;
+
+    domain domainAux;
+    domainAux.max= 100;
+    domainAux.min = -100;
+    population = malloc(sizeof(population));
+    population->individuos = generate_population(124, 10, domainAux, 3);
```

Comment:
generate_island

Reply:
Só subtituir por isso:

```suggestion
   population = generate_island(1, 124, 10, domainAux, 3);

```